docs/README.md

```diff
@@ -5,9 +5,9 @@
 
 # Introduction
 
-**Pipeline abstractions for deep learning**
+# Pipeline Abstractions for Deep Learning
```

Comment:
Could you undo this? The `**` puts `Pipeline Abstractions for Deep Learning` under the Introduction tab on our docs page. Otherwise we get this
![Screenshot 2021-12-15 at 14 10 23](https://user-images.githubusercontent.com/10526848/146192393-576ba412-e64a-4d58-8490-d763c3b45a86.png)

Reply:
hm, okay, strange.